README.md

```diff
@@ -277,6 +277,54 @@ response = stripe.raw_request(
 deserialized_resp = stripe.deserialize(response)
 ```
 
+### Async
+
+Asynchronous versions of most request-making methods are available by suffixing the method name
```

Comment:
why most? should we clarify what's missing and whether customers should expect it in the future?

Reply:
Note about `.save` a little below